gulp-tasks/utils/uglify-name-cache-helper.js

```diff
@@ -0,0 +1,18 @@
+const fs = require('fs-extra');
+
+const constants = require('./constants');
+
+module.exports = {
```

Comment:
Adding a comment here along the lines of:

To enable uglify to minify the async and await generators and the babel helpers, we need to share the cache names between the individual module builds.

This file will stashes the cache names locally and saves the cache names after each module build.